scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -89,8 +86,8 @@ public static BuildTargetResult RestoreTestAssetProjects(BuildTargetContext c)
             CleanNuGetTempCache();
 
             var dotnet = DotNetCli.Stage2;
-
-            dotnet.Restore("--verbosity", "verbose", "--disable-parallel", "--infer-runtimes", "--fallbacksource", Dirs.TestPackages)
+            dotnet.Restore("--verbosity", "verbose", "--disable-parallel", "--infer-runtimes",
```

Comment:
Usually, when I break some parameter down to the next line, I put one parameter per line. It makes it easier to read.